.changeset/pretty-ladybugs-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': patch
```

Comment:
Since we're changing the API in a backwards compatible way, should this be a minor? Same for `@backstage/plugin-techdocs`.

Reply:
I had thought `minor` was reserved for breaking changes - I set it based off the contribution guide but I am happy to convert it to a `minor` bump if that's more appropriate. 

Have the guidelines changed since plugins passed the `1.0.0` mark?

Reply:
Yeah for `1.x.y`, backwards compatible API changes should be considered considered minors. I think this example applies to this change: https://github.com/backstage/backstage/blob/master/REVIEWING.md#changeset-example

I assume you refer to this part of `CONTRIBUTING.md`
>Select impact of change that you're introducing, using minor for breaking changes and patch otherwise. We do not use major changes while packages are at version 0.x.

And I think it's missleading. My guess is that it was written at a time when most packages were at `0.x.y`. It should be changed. I'll look into it.

Reply:
Ah! I had missed that example, that clears things up - thanks for sharing. I'll update the changesets to minor then.

Reply:
When packages are `0.x.x` any breaking changes are `minor` and other changes are `patch`. `tech-docs` should be `minor` and the `core-components` should be `patch`. It's so that we don't have `v31.0.0` until we get a stable API for these things :)

Reply:
Sorry if theres some confusion maybe we can clarify it a little better in the docs :)

Reply:
Makes sense - I will adjust the changeset for core-components.

Reply:
Ah cool, didn't realize core-components was `0.x.y` 👍 👍